__tests__/api.test.js

```diff
@@ -127,6 +127,15 @@ describe('GET /api/articles/:article_id', () => {
         expect(body.message).toBe('bad request');
         });
     })
+    test('the response objects has a comment_count property, which is the total number of comments for that article', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            const commentCount = body.article.comment_count
```

Comment:
Please check the other properties as well and the exact article id to make sure it's the right article with all it's properties present.

Reply:
i was going to, but noticed my first test tests for those so deleted them
will add them back in
thanks

Reply:
done

Reply:
apologies
i see
do this testing of properties as it could be a different article number